x-pack/legacy/plugins/maps/public/connected_components/layer_panel/join_editor/resources/where_expression.js

```diff
@@ -78,28 +80,35 @@ export class WhereExpression extends Component {
               defaultMessage="Use a query to narrow right source."
             />
           </EuiFormHelpText>
-          <SearchBar
-            uiSettings={settings}
-            showFilterBar={false}
-            showDatePicker={false}
-            showQueryInput={true}
-            query={whereQuery ? whereQuery : { language: settings.get('search:queryLanguage'), query: '' }}
-            onQuerySubmit={this._onQueryChange}
-            appName="maps"
-            indexPatterns={[indexPattern]}
-            store={localStorage}
-            customSubmitButton={
-              <EuiButton
-                fill
-                data-test-subj="mapWhereFilterEditorSubmitButton"
-              >
-                <FormattedMessage
-                  id="xpack.maps.layerPanel.whereExpression.queryBarSubmitButtonLabel"
-                  defaultMessage="Set filter"
-                />
-              </EuiButton>
-            }
-          />
+
+          <KibanaContextProvider
```

Comment:
Here we could use connected `<SearchBar>` component, too.

Reply:
In the next PR!